modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultCodegen.java

```diff
@@ -137,6 +137,7 @@ public String escapeText(String input) {
         if (input != null) {
             input = input.trim();
             String output = input.replaceAll("\n", "\\\\n");
+            output = output.replace("\r", "\\r");
```

Comment:
@artem-dudarev thanks for the PR. Should it be replaced with "\\\\r" similar to the replaceAll for \n and \" (with \\\n and \\\" respectively)?